core/server/master/src/main/java/alluxio/master/file/meta/InodeView.java

```diff
@@ -103,12 +103,18 @@
   boolean isDeleted();
 
   /**
-   * @return true if the inode is a directory, false otherwise
+   * Checks whether the inode is a directory or not.
+   *
+   * @return  a boolean value representing whether the inode
+   *          is a directory
    */
   boolean isDirectory();
 
   /**
-   * @return true if the inode is a file, false otherwise
+   * Checks whether the inode is a file.
+   *
+   * @return  a boolean representing whether the {@link Inode}
+   *          implementing this {@link InodeView} is a file
```

Comment:
You can write a better description. Say that return False if is not a file and True if it's.